### ISSUE TITLE
Search across notes

### ISSUE THREAD

**Issue by eljefuri** [lainsce/notejot#96]
The [Stickies](https://en.wikipedia.org/wiki/Stickies_(Apple)) application on Mac OS X featured the ability to search across all notes, which was very useful. One simply pressed ⌘F and a pop-up appeared. See [here](https://youtu.be/M1DscVsO2uE?t=299) for a video of the feature in action. This would be hugely useful for Notejot as well.

**Comment by lainsce** [comment]
I might need to think about how to best show this to the user.

**Comment by eljefuri** [comment]
I should bloody well hope so, especially before implementing it. :) Thank you very much for considering this feature request @lainsce

**Comment by lainsce** [comment]
After much planning and researching and everything in-between, I've decided that this feature would be very intensely difficult to implement, due to how the app handles notes. I'm sorry, but it's not feasible to implement this. :disappointed:

**Comment by eljefuri** [comment]
Fair enough, just a nice-to-have.

**Comment by o-alquimista** [comment]
I often find myself pressing `Ctrl + F` to search within the current note. Would it be any easier to implement search functionality within a single note?

**Comment by lainsce** [comment]
If you need search within a note, isn't that note perhaps a bit wordy? Try finding ways to segment notes into new notes instead 😅

**Comment by o-alquimista** [comment]
I have one note for each project I'm working on. They get large, I know. It's inevitable. I have an avalanche of ideas. But sometimes I want to make sure I'm not including the same idea twice. It's easy to miss an item you have already included.

This is an illustration of my usual list of ideas and bugfix reminders:
```
- idea 1 idea 1 idea 1 idea 1
idea 1 idea 1.
- idea 2 idea 2 idea 2 idea 2
idea 2 idea 2 idea 2 idea 2
idea 2 idea 2.
- idea 3 idea 3 idea 3 idea 3
```

They are usually longer than the note window, so they wrap like that. This, combined with the small line height, makes it hard to find something by skimming.

How about supporting bulleted lists with a slightly larger line height and a different text wrapping strategy?
```
•  idea 1 idea 1 idea 1 idea 1
   idea 1 idea 1.

•  idea 2 idea 2 idea 2 idea 2
   idea 2 idea 2 idea 2 idea 2
   idea 2 idea 2.

•  idea 3 idea 3 idea 3 idea 3
```

**Comment by lainsce** [comment]
Unfortunately, I can't control much for the word wrap. I have three modes: Wrap by letter, word, or words and letters. Those don't seem to be what you want. :woman_shrugging: 
As for the bulleted list, I also can't do that.
